crates/bevy_winit/src/winit_windows.rs

```diff
@@ -1,21 +1,21 @@
 use crate::converters::convert_cursor_grab_mode;
-use bevy_math::{DVec2, IVec2};
-use bevy_utils::HashMap;
+use bevy_ecs::entity::Entity;
+
+use bevy_utils::{tracing::warn, HashMap};
 use bevy_window::{
-    CursorGrabMode, MonitorSelection, RawHandleWrapper, Window, WindowDescriptor, WindowId,
-    WindowMode,
+    CursorGrabMode, WindowComponentsItem, WindowMode, WindowPosition, WindowResolution,
 };
-use raw_window_handle::{HasRawDisplayHandle, HasRawWindowHandle};
+
 use winit::{
-    dpi::{LogicalPosition, LogicalSize, PhysicalPosition, PhysicalSize},
-    window::Fullscreen,
+    dpi::{LogicalPosition, LogicalSize, PhysicalPosition},
+    monitor::MonitorHandle,
 };
 
 #[derive(Debug, Default)]
 pub struct WinitWindows {
     pub windows: HashMap<winit::window::WindowId, winit::window::Window>,
-    pub window_id_to_winit: HashMap<WindowId, winit::window::WindowId>,
-    pub winit_to_window_id: HashMap<winit::window::WindowId, WindowId>,
+    pub window_id_to_winit: HashMap<Entity, winit::window::WindowId>,
```

Comment:
Suggest we rename this as we are no longer using `WindowId`
```suggestion
    pub entity_to_window: HashMap<Entity, winit::window::WindowId>,
```